### PR TITLE
Feature/sdl 0269 New vehicle data ClimateData

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
Guidelines updates according to the proposal [SDL-0269 New vehicle data ClimateData](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0269-New-vehicle-data-ClimateData.md) 

This PR is **ready** for review.

### Summary

Expand `VehicleDataType` enum with new param
Add new struct `ClimateData`
Add `climateData` to SubscribeVehicleData, UnsubscribeVehicleData, GetVehicleData, OnVehicleData RPCs

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

**Review by atiwari9 — APPROVED** [review]

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -1179,3 +1179,12 @@ There are no defined parameters for this struct
 |:---|:---|:--------|:---------|:----------|
 |location|Common.Grid|true|||
 |state|Common.WindowState|true|||
+
+### ClimateData 
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|externalTemperature|[Common.Temperature]()|false||The external temperature in degrees celsius|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1187]:
```suggestion
|externalTemperature|Common.Temperature|false||The external temperature in degrees celsius|
```

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -1179,3 +1179,12 @@ There are no defined parameters for this struct
...
+|cabinTemperature| [Common.Temperature]|false||Internal ambient cabin temperature in degrees celsius|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1188]:
```suggestion
|cabinTemperature|Common.Temperature|false||Internal ambient cabin temperature in degrees celsius|
```

File: docs/Common/Structs/index.md [hunk 3]
```diff
@@ -1179,3 +1179,12 @@ There are no defined parameters for this struct
...
+|atmosphericPressure|float|false|minvalue="0" maxvalue="2000"|Current atmospheric pressure in mBar|
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1189]:
```suggestion
|atmosphericPressure|float|false|minvalue: 0<br>maxvalue: 2000|Current atmospheric pressure in mBar|
```

**Review by ShobhitAd — CHANGES_REQUESTED** [review]

**Review by ShobhitAd — APPROVED** [review]

**Comment by ShobhitAd** [comment]
@KhrystynaDubovyk Please resolve merge conflicts

**Comment by KhrystynaDubovyk** [comment]
@ShobhitAd, done in  https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/234/commits/77d384c01961bd5204731d749273a1754745150b

File: docs/Common/Structs/index.md [hunk 4]
```diff
@@ -1227,3 +1227,10 @@ There are no defined parameters for this struct
 |seatsOccupied|Common.SeatStatus|false|array: true<br>minsize: 0<br>maxsize: 100|Seat status array containing location and whether the seats are occupied.|
 |seatsBelted|Common.SeatStatus|false|array: true<br>minsize: 0<br>maxsize: 100|Seat status array containing location and whether the seats are belted.|
 
+### ClimateData
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1230]:
```suggestion
### ClimateData
```

**Review by ShobhitAd — CHANGES_REQUESTED** [review]

**Review by ShobhitAd — APPROVED** [review]

**Pull request merged**